diff -urN tea-3.3.oryg/Makefile.am tea-3.3/Makefile.am
--- tea-3.3.oryg/Makefile.am	2004-09-24 18:23:59.160021352 +0200
+++ tea-3.3/Makefile.am	2004-09-24 18:58:00.245729160 +0200
@@ -4,25 +4,12 @@
 
 EXTRA_DIST = \
 	autogen.sh \
-	FAQ \
-	BUGS \
-	tea_icon_v2.png \
-	doc
+	tea_icon_v2.png
 
      
 
 install-data-local:
 	@$(NORMAL_INSTALL)
-	$(mkinstalldirs) $(DESTDIR)$(pkgdatadir)/doc; \
-	cp -r $(srcdir)/doc $(DESTDIR)$(pkgdatadir); \
-	cp FAQ $(DESTDIR)$(pkgdatadir)/doc; \
-	cp AUTHORS $(DESTDIR)$(pkgdatadir)/doc; \
-	cp ChangeLog $(DESTDIR)$(pkgdatadir)/doc; \
-	cp README $(DESTDIR)$(pkgdatadir)/doc; \
-	cp NEWS $(DESTDIR)$(pkgdatadir)/doc; \
-	cp COPYING $(DESTDIR)$(pkgdatadir)/doc; \
-	cp TODO $(DESTDIR)$(pkgdatadir)/doc; \
-	cp BUGS $(DESTDIR)$(pkgdatadir)/doc; \
 	if test -d $(srcdir)/pixmaps; then \
 	  $(mkinstalldirs) $(DESTDIR)$(pkgdatadir)/pixmaps; \
 	  for pixmap in $(srcdir)/pixmaps/*; do \
